src/include/sof/schedule/ll_schedule.h

```diff
@@ -41,6 +41,7 @@ int schedule_task_init_ll(struct task *task,
 			  const struct sof_uuid_entry *uid, uint16_t type,
 			  uint16_t priority, enum task_state (*run)(void *data),
 			  void *data, uint16_t core, uint32_t flags);
+void schedule_ll_run_tasks(void);
```

Comment:
this is only needed for LIBRARY builds, right? Can we declare it somewhere where it's only visible to such builds?